android-wsbridge/app/src/main/java/com/dji/wsbridge/BridgeActivity.java

```diff
@@ -317,12 +321,12 @@ private void refreshRunners() {
                 deviceToWSRunner = new StreamRunner(wsInputStream, usbOutputStream, "Bridge to USB");
                 wsToDeviceRunner = new StreamRunner(usbInputStream, wsOutputStream, "USB to Bridge");
                 try {
-                    Crashlytics.log("Device to WS Runner alive "+ deviceToWSRunner.isAlive());
-                    Crashlytics.log("WS to Device Runner alive "+ wsToDeviceRunner.isAlive());
+                    logToFirebase("Device to WS Runner alive " + deviceToWSRunner.isAlive());
```

Comment:
Is it ok to do Firebase call without initializing? For non internal builds

Reply:
The utility function is checking for initialization.